### PR TITLE
Mass import feature

### PR DESCRIPTION
**Pull request by ZhuLeYao**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/jobs/ImportDeliveryJobCommand.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        List<DeliveryJob> listOfAddDeliveryJob;
+        listOfAddDeliveryJob = ImportDeliveryJobCommandParser.parse(toAdd);
+
+        for (int i = 0; i < listOfAddDeliveryJob.size(); i++) {
+            model.addDeliveryJob(listOfAddDeliveryJob.get(i));
+        }
+
+        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+    }
+}
```
Comment (dohaduong) [inline src/main/java/seedu/address/logic/commands/jobs/ImportDeliveryJobCommand.java:47]: You may want to add EOF blank line heree, i think something's wrong and the checks couldn't run but it'll prompt the error again

File: src/main/java/seedu/address/logic/parser/jobs/ImportDeliveryJobCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            String ear = arrOfStr[4];
+
+            DeliveryJob job = new DeliveryJob(rid, sid, ded, des, ear, "");
+
+            listOfAddDeliveryJob.add(job);
+        }
+
+        return listOfAddDeliveryJob;
+    }
+}
```
Comment (dohaduong) [inline src/main/java/seedu/address/logic/parser/jobs/ImportDeliveryJobCommandParser.java:50]: EOF blank line too!

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 3]
```diff
@@ -508,6 +535,8 @@ private CommandResult executeCommand(String commandText) throws CommandException
             logger.info("Invalid command: " + commandText);
             resultDisplay.setFeedbackToUser(e.getMessage());
             throw e;
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e);
```
Comment (dohaduong) [inline src/main/java/seedu/address/ui/MainWindow.java:539]: ooh wait this part do we need to prompt users that file does not exist (like through resultDisplay maybe)

File: src/main/java/seedu/address/ui/StatisticsWindow.java [hunk 4]
```diff
@@ -194,6 +195,8 @@ private CommandResult executeCommand(String commandText) throws CommandException
             resultDisplay.setFeedbackToUser(e.getMessage());
             //resultDisplay.setFeedbackToUser(e.getMessage());
             throw e;
+        } catch (FileNotFoundException e) {
```
Comment (dohaduong) [inline src/main/java/seedu/address/ui/StatisticsWindow.java:198]: same here, should we setFeedbackToUser?

**Review by dohaduong — CHANGES_REQUESTED** [review]
lgtm, but can add the EOF blank line i think :) just to pass the checks when they re run

File: src/main/java/seedu/address/logic/commands/jobs/ImportDeliveryJobCommand.java (hunk 1 above)
Comment (ZhuLeYao) [reply src/main/java/seedu/address/logic/commands/jobs/ImportDeliveryJobCommand.java:53]: okie

File: src/main/java/seedu/address/logic/parser/jobs/ImportDeliveryJobCommandParser.java (hunk 2 above)
Comment (ZhuLeYao) [reply src/main/java/seedu/address/logic/parser/jobs/ImportDeliveryJobCommandParser.java:54]: done!

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 3 above)
Comment (ZhuLeYao) [reply src/main/java/seedu/address/ui/MainWindow.java]: updated all similar areas

File: src/main/java/seedu/address/ui/StatisticsWindow.java (hunk 4 above)
Comment (ZhuLeYao) [reply src/main/java/seedu/address/ui/StatisticsWindow.java:199]: updated all similar areas

**Review by dohaduong — APPROVED** [review]
looks good!

**Pull request merged**